src/Select/Select.tsx

```diff
@@ -221,6 +221,7 @@ const ReactSelect = forwardRef(
             aria-label={ariaLabel}
             filterOption={filterOption}
             closeMenuOnSelect={closeMenuOnSelect}
+            {...props}
```

Comment:
Should we spread first, and then apply the other props? How do we want to handle prop collisions? do we want `props` to take precedence?

Reply:
Make sense!